automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/security/AuthDomainSpec.scala

```diff
@@ -529,7 +529,8 @@ class AuthDomainSpec extends FreeSpec /*with ParallelTestExecution*/ with Matche
       }
     }
 
-    "+ project owner, + group member, create workspace, - project owner" in withWebDriver { implicit driver =>
+    // Ignored as of 2/16/2018 - ticket GAWB-TBD
```

Comment:
Let's make the ticket and update this reference before disabling.

Reply:
Created and added